requirements.txt

```
@@ -0,0 +1,7 @@
+akari-client[depthai]
+akari-proto
+opencv-python
+mediapipe
+depthai
```

```suggestion
depthai==2.30.0
```

9/8にdepthai3.0.0がリリースされ、これにより既存の実装は動作しなくなりました…。
従ってdepthaiのバージョンを2に固定する必要があります。